flake.nix

```diff
@@ -72,13 +72,14 @@
         buildInputs = oldAttrs.buildInputs ++ [pkgs.pkg-config pkgs.libusb1];
         # run actual build
         buildPhase = ''
-          yarn build
+          yarn workspace @cardano-sdk/cardano-services run build
         '';
         # override installPhase to only install what's necessary
         installPhase = ''
           runHook preInstall
 
           mkdir -p $out/libexec/$sourceRoot $out/bin
+          yarn workspace @cardano-sdk/cardano-services-client run build:version
```

Comment:
Shouldn't this be in `buildPhase`? :thinking:

Reply:
I was getting some read-only write error, not sure why. It works here thus I did not bother looking into it